### PR TITLE
chore: fix comment

### LINKED ISSUE #97: TRANSFER_ITEM_SEEKER Comments don't match variable names
TRANSFER_ITEM_SEEKER is an action used in a crafting extension so this is some user facing docs.
It's quite confusing as to what the various slot and bag indexes refer to.

https://github.com/playmint/ds/blob/593b009b899af0438fad6f4352a6c555fec359f6/contracts/src/actions/Actions.sol#LL19

### PR DESCRIPTION
**Pull request by chrisfarms**
closes #97

### CODE REVIEW

File: contracts/src/actions/Actions.sol [hunk 1]
```diff
@@ -16,7 +16,7 @@ interface Actions {
     // to scout location
     function SCOUT_SEEKER(uint32 sid, int16 q, int16 r, int16 s) external;
 
-    // transfers a balance of whatever is in fromBag/fromSlot into toBag/toSlot
+    // transfers a balance between item slots
```
Comment (5p0rt5BEArD) [inline contracts/src/actions/Actions.sol:19]: I would still have to go and read the code to see what params do. Its not clear what the size 2 arrays are.

how about:

// transfer a qty of items from itemSlot[0] in equipees[0]'s equipSlots[0] bag, to itemSlot[1] in  equipees[1]'s equipSlots[1] bag

**Review by 5p0rt5BEArD — CHANGES_REQUESTED** [review]
I appreciate this is a long winded way to change a comment - I was more dipping my toes in the issue system when I wrote the issue. But as a user it would have been useful to have a more explicit comment.

File: contracts/src/actions/Actions.sol (hunk 1 above)
Comment (chrisfarms) [reply contracts/src/actions/Actions.sol]: done!

**Comment by chrisfarms** [comment]
hah yeah def quicker to press the edit button that raise an issue 😆  ... but good practice

obviously the _real_ issue here is "have some proper documentation with examples" as I think similar comments could be made about pretty much every action being rather mysterious and opaque 😄

**Review by 5p0rt5BEArD — APPROVED** [review]

**Pull request merged**